kubeflow-pipelines.advisories.yaml

```diff
@@ -192,11 +288,77 @@ advisories:
           note: Vulnerable code is part of the Kubernetes API server and was also mark NOT_IMPORTABLE in Golang vulndb
 
   - id: CVE-2023-2728
-    aliases:
-      - GHSA-cgcv-5272-97pr
     events:
       - timestamp: 2023-09-09T17:14:39Z
         type: false-positive-determination
         data:
           type: vulnerable-code-not-included-in-package
           note: Vulnerable code is part of the Kubernetes ServiceAccount admission plugin which is not importable and also code was marked NOT_IMPORTABLE in Golang vulndb
+
+  - id: CVE-2023-28155
+    aliases:
+      - GHSA-p8p7-x288-28g6
+    events:
+      - timestamp: 2023-11-01T07:12:35Z
+        type: fix-not-planned
+        data:
+          note: cannot fix the CVE because there were no patched versions since the 'request' package is no longer supported by the maintainer.
+
+  - id: CVE-2023-34104
+    aliases:
+      - GHSA-6w63-h3fj-q4vw
+    events:
+      - timestamp: 2023-11-01T06:56:37Z
+        type: pending-upstream-fix
+        data:
+          note: fast-xml-parser should be upgraded to 4.2.4
+
+  - id: CVE-2023-43804
+    events:
+      - timestamp: 2023-11-01T11:36:40Z
+        type: pending-upstream-fix
+        data:
+          note: urllib3 should be upgraded to 1.26.17
+
+  - id: CVE-2023-45803
+    events:
+      - timestamp: 2023-11-01T11:36:15Z
+        type: pending-upstream-fix
+        data:
+          note: urllib3 should be upgraded to 1.26.18
+
+  - id: CVE-2023-46233
+    events:
+      - timestamp: 2023-11-01T06:38:31Z
+        type: pending-upstream-fix
+        data:
+          note: crypto-js should be upgraded to 4.2.0
+
+  - id: GHSA-2jcg-qqmg-46q6
+    events:
+      - timestamp: 2023-11-01T07:13:06Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: This is test code that is never used in production. To get more detail about the false-positive determination, please check https://github.com/browserify/resolve/issues/288
+
+  - id: GHSA-5rrq-pxf6-6jx5
+    events:
+      - timestamp: 2023-11-01T07:05:18Z
+        type: pending-upstream-fix
+        data:
+          note: '''node-forge'' is a transitive dependency of ''google-p12-pem'', which is pinned to an old release. Attempts to patch have been unsuccessful, as the new releases introduced a lot of changes. Waiting on upstream to address.'
+
+  - id: GHSA-gf8q-jrpm-jvxq
+    events:
+      - timestamp: 2023-11-01T07:07:11Z
+        type: pending-upstream-fix
+        data:
+          note: '''node-forge'' is a transitive dependency of ''google-p12-pem'', which is pinned to an old release. Attempts to patch have been unsuccessful, as the new releases introduced a lot of changes. Waiting on upstream to address.'
+
+  - id: GHSA-x3cc-x39p-42qx
+    events:
+      - timestamp: 2023-11-01T07:03:32Z
+        type: pending-upstream-fix
+        data:
+          note: fast-xml-parser should be upgraded to 4.1.2
```

Comment:
This comment will need updated as well

Reply:
fast-xml-parser this one and this crypto-js are both main dependencies. So, I tried to bump their version through the `overrides` section in the package.json but it didn't work so I left the note as `fast-xml-parser should be upgraded to <version>`.